src/redux/reducer.js

```diff
@@ -1,43 +1,15 @@
 var combineReducers = require('redux').combineReducers;
 
-var actionTypes = require('./actions.js').types;
-
-
-var sessionReducer = function (state, action) {
-    // Reducer for handling changes to session state
-    if (typeof state === 'undefined') {
-        state = {};
-    }
-    switch (action.type) {
-    case actionTypes.SET_SESSION:
-        return action.session;
-    case actionTypes.SET_SESSION_ERROR:
-        // TODO: do something with action.error
-        return state;
-    default:
-        return state;
-    }
-};
-
-var tokenReducer = function (state, action) {
-    // Reducer for updating the api token
-    if (typeof state === 'undefined') {
-        state = '';
-    }
-    switch (action.type) {
-    case actionTypes.SET_TOKEN:
-        return action.token;
-    case actionTypes.SET_TOKEN_ERROR:
-        // TODO: do something with the error
-        return state;
-    default:
-        return state;
-    }
-};
+var authReducers = require('./auth.js');
+var conferenceScheduleReducers = require('./conference-schedule.js');
+var conferenceDetailsReducers = require('./conference-details.js');
 
 var appReducer = combineReducers({
-    session: sessionReducer,
-    token: tokenReducer
+    session: authReducers.sessionReducer,
+    token: authReducers.tokenReducer,
+    day: conferenceScheduleReducers.dayReducer,
+    schedule: conferenceScheduleReducers.scheduleReducer,
+    details: conferenceDetailsReducers.detailsReducer
```

Comment:
I like the organization changes you made for redux.  I think we could benefit from following the [ducks](https://medium.com/@scbarrus/the-ducks-file-structure-for-redux-d63c41b7035c#.j91k3mtyl) method more, and isolate each module to one reducer per file.  That way when you combine reducers here, the state can just be the name of the reducer, so it's clear what part of the state comes from where.  Right now `day`, `schedule`, and `details` are ambiguous.

So I suggest breaking up authReducers into token and session, and conferenceScheduleReducers int conferenceDay and conferenceSchedule.  Since in ducks, they recommend you export the reducer, and put the actions as properties of the reducer, for example:
```javascript
var TYPES = keyMirror({
//...
});
var reducer = function (state, action) {
//...
}
reducer.fooAction = function () {/* ... */}
reducer.thunkifiedAction = function () {
    return function (dispatch) {
        /* ... */
        // This needs to dispatch another action
        return dispatch(reducer.fooAction());
    }
}
// ...
module.exports = reducer;
```

and then `combineReducers` would be a simple map:
```javascript
var appReducer = combineReducers({
    session: session,
    token: token,
    conferenceDay: conferenceDay,
    conferenceSchedule: conferenceSchedule,
    conferenceDetails: conferenceDetails
});

// or in ES6
var appReducer = combineReducers({
    token, session, conferenceDay, conferenceSchedule, conferenceDetails
});
```

I think this would keep things more organized since then the state is inherently namespaced, and the name of the state implies the name of the module, so that's where you would go to get actions, or to reference for how the reducer uses those actions.

...

Alternatively I would just suggest being more specific in your naming of `day`, `schedule`, and `details` here :)